File: cpp_solution/Forloop.cpp
// TUF QUESTION 9:
//Given two integers low and high, return the sum of all integers from low to high inclusive.
// prctice solution:
#include <iostream>
using namespace std;
int main(){
    int low= 3;
    int high = 7;
    int sum=0 ;
    for(int i=low;i<=high;i++){
        int digit = i;
        sum = digit + sum;
        }
    cout<<"sum :"<<sum;
    return 0;
    }
    
//TUF SOLUTION:
class Solution {
public:
    int forLoop(int low, int high) {
        int Sum=0;
        for(int i=low;i<=high;i++){
            int Digit = i;
            Sum= Digit + Sum; 
        }
        return Sum;
    }
};
    

File: Python_solutions/things to know in python/forloop.py
#TUF QUESTION 9:
#Given two integers low and high, return the sum of all integers from low to high inclusive.
# prctice solution:
low=3
high=7
sum=0
for i in range(low, high+1):
    digit =i
    sum+=digit
print("sum :", sum)

# TUF SOLUTION
class Solution:
    def forLoop(self, low : int, high : int) -> int:
        sum=0
        for i in range(low, high+1):
            current_digit= i
            sum+=current_digit
        return sum
